config/PdoStatement.stub

```diff
@@ -19,7 +19,7 @@ class PDOStatement implements Traversable, IteratorAggregate
     public function fetchObject($class = \stdclass::class, array $ctorArgs = array()) {}
     
     /**
-     * @return array{name: string, table: string, native_type: string, len: int, flags: array<int, string>, precision: int<0, max>, pdo_type: PDO::PARAM_* }|false
+     * @return array{name: string, table?: string, native_type: string, len: int, flags: array<int, string>, precision: int<0, max>, pdo_type: PDO::PARAM_* }|false
```

Comment:
why do we need this change?

Reply:
I changed it to support this line:
https://github.com/staabm/phpstan-dba/pull/328/files#diff-0662669ff7c8340cd0a9b36fb64fa2f8cb6790c08065e8731d477e26e77ba4ffR73

For some queries, I think it was for example when you use e.g. `select count(*) ..` or similar, where it's not a column reference the `table` key is not present in the column meta. Can't recollect exactly..